skfuzzy/control/fuzzyvariable.py

```diff
@@ -31,7 +69,7 @@ class FuzzyVariable(object):
     This class is designed as the base class underlying the Antecedent and
     Consequent classes, not for individual use.
     """
-    def __init__(self, universe, label):
+    def __init__(self, universe, label, defuzzy_method='centroid'):
```

Comment:
`defuzz_method` or `defuzzify_method` would be more consistent with existing names in the package. Though if there is a strong case for 'defuzzy' we could deprecate those old names.